src/traces/sankey/render.js

```diff
@@ -616,6 +686,7 @@ module.exports = function(svg, calcData, layout, callbacks) {
           .attr('d', linkPath())
           .call(attachPointerEvents, sankey, callbacks.linkEvents);
 
+
```

Comment:
:zap: no need to commit this newline.

Reply:
Done in f628b549433df3f86244125a2e66929781929f69